src/main/java/ch/njol/skript/expressions/ExprVectorRandom.java

```diff
@@ -57,8 +54,8 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	}
 
 	@Override
-	protected Vector[] get(Event e) {
-		return CollectionUtils.array(new Vector(randomSignedDouble(), randomSignedDouble(), randomSignedDouble()));
+	protected Vector[] get(Event event) {
+		return CollectionUtils.array(new Vector(randomSignedDouble(), randomSignedDouble(), randomSignedDouble()).normalize());
```

Comment:
```suggestion
		// Generating uniform random numbers leads to bias towards the corners of the cube. Gaussian distribution is radially symmetric, so it's properly uniform.
		return CollectionUtils.array(new Vector(random.nextGaussian(), random.nextGaussian(), random.nextGaussian()).normalize());
```

Reply:
Permission was granted to change the way vectors are generated within this PR scope.